assets/src/scripts/components/map/index.js

```diff
@@ -8,8 +8,15 @@ import { FLOOD_EXTENTS_ENDPOINT, FLOOD_BREACH_ENDPOINT, FLOOD_LEVEE_ENDPOINT } f
 import { hasFloodData, getFloodExtent, getFloodStageHeight } from '../../selectors/flood-data-selector';
 import { Actions } from '../../store';
 import { floodSlider } from './flood-slider';
-import { createLegendControl, createFIMLegend } from './legend';
-import { addNldi } from './nldiMapping';
+import { createLegendControl, createFIMLegend, createNldiLegend } from './legend';
+import { addNldiLayers} from './nldiMapping';
+import {
+    hasNldiData,
```

Comment:
Nit. Our pattern has been to put all on the same line. If more than one line is required, then continue on to the next line (see https://github.com/usgs/waterdataui/blob/master/assets/src/scripts/components/hydrograph/index.js for examples)